### ISSUE TITLE
Show a toast notification if network access is restricted for an app 

### ISSUE THREAD

**Issue by aspasskaya** [AdguardTeam/AdguardForAndroid#2192]
### Steps to reproduce

1. Disable the option in settings: Apps management -> Apps and Statistics -> App Details 'Chrome' -> Wi-Fi data
2. Open the app 'Chrome' and go to lenta.ru. Result: The web-site doesn't working and and the toast notification appeared 'Chrome blocked [Customize the AdGuard firewall]
3. Tap to the Notification button and enable the option Wi-Fi data
4. Disable the option Wi-Fi data

### Expected behavior

When working with the app. "Chrome' and when you enable or disable the option 'Wi-Fi data', the blocking toast notification must always appears. 

### Actual behavior

When you enable or disable the option 'Wi-Fi data', the blocking toast notification appears only once.

**Comment by ameshkov** [comment]
Fixing this is quite complicated and we'd better not do it right now for a low-priority bug

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdguardForAndroid/issues/3379
